docker-compose-dev.yml

```diff
@@ -16,13 +16,12 @@ services:
       ENABLE_RELOAD: "1"
       SKIP_DEPLOY: "${SKIP_DEPLOY}"
       DATAVERSE_JSF_REFRESH_PERIOD: "1"
-      DATAVERSE_FEATURE_API_BEARER_AUTH: "1"
       DATAVERSE_MAIL_SYSTEM_EMAIL: "dataverse@localhost"
       DATAVERSE_MAIL_MTA_HOST: "smtp"
-      DATAVERSE_AUTH_OIDC_ENABLED: "1"
-      DATAVERSE_AUTH_OIDC_CLIENT_ID: test
-      DATAVERSE_AUTH_OIDC_CLIENT_SECRET: 94XHrfNRwXsjqTqApRrwWmhDLDHpIYV8
-      DATAVERSE_AUTH_OIDC_AUTH_SERVER_URL: http://keycloak.mydomain.com:8090/realms/test
```

Comment:
Why are we removing OIDC from the dev compose file? Before I jumped on this PR I re-tested it on the develop branch and it works fine. Screenshots below. In this PR the OIDC button is gone. 😢 

![Screenshot 2024-10-03 at 2 05 10 PM](https://github.com/user-attachments/assets/c08ef40b-aa13-4b92-b6a2-d70f2cb5e544)
![Screenshot 2024-10-03 at 2 05 22 PM](https://github.com/user-attachments/assets/8dd79b30-d071-479d-aaf9-1e02d8b0ada2)
![Screenshot 2024-10-03 at 2 05 29 PM](https://github.com/user-attachments/assets/e11751f3-13cf-4d0e-b8d4-78ca4681afa7)